app/app/sitemaps.py

```diff
@@ -22,6 +24,20 @@ def location(self, item):
         return reverse(item)
 
 
+class GrantSitemap(Sitemap):
+    changefreq = "daily"
+    priority = 0.9
+
+    def items(self):
+        return Grant.objects.all()
+
+    def lastmod(self, obj):
+        return obj.modified_on
+
+    def location(self, item):
+        return f"/grants/{str(item.id)}/{item.slug}"
```

Comment:
You can eliminate the `str()` here.  The f-string formats the value to `str` automatically by default.

Reply:
Ah, sure, didn't really figured out all of em, removing it...